dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/companies/CompanyAssociatedData.kt

```diff
@@ -17,5 +18,6 @@ data class CompanyAssociatedData<T> (
     val reportingPeriod: String,
 
     @field:JsonProperty(required = true)
+    @field:Valid
```

Comment:
What does this Valid do?

Reply:
@teloojustus this @field:Valid is responsible for using validation on the corresponding field, correct?

Reply:
ok